cockroachcloud/learn-about-pricing.md

```diff
@@ -9,42 +9,30 @@ This page describes how charges accumulate in {{ site.data.products.serverless }
 
 ## Pricing overview
 
-With {{ site.data.products.serverless }}, you are charged for the storage and activity of your cluster. Cluster storage is measured in GiB per month, based on the total volume of storage used over the billing period. Cluster activity, including SQL queries, bulk operations, and background jobs, is measured in [Request Units](learn-about-request-units.html), or RUs. A Request Unit is an abstracted metric that represents the size and complexity of requests made to your cluster. Note that a single activity or request could cost more than 1 Request Unit. Request Unit consumption scales to zero when your cluster has no activity, so you will only be charged for what you use.
+With {{ site.data.products.serverless }}, you are charged for the storage and activity of your cluster. Cluster storage is measured in GiB and based on the total volume of storage used over the billing period. Cluster activity, including SQL queries, bulk operations, and background jobs, is measured in [Request Units](learn-about-request-units.html), or RUs. A Request Unit is an abstracted metric that represents the size and complexity of requests made to your cluster. Note that a single activity or request could cost more than 1 Request Unit. Request Unit consumption scales to zero when your cluster has no activity, so you will only be charged for what you use.
 
 RU and storage consumption is prorated at the following prices:
 
   Unit                    | Cost
   ------------------------|------
-  10M Request Units       | $1.00
-  1 GiB storage           | $1.00
+  1M Request Units        | $0.20
```

Comment:
@andy-kimball @gemma-shay are we confirmed that we want to shift from "10M" to "1M" as the pricing unit for RU's?

Reply:
That was Emily's call originally. I think 1M is a nicer unit to use in mental calculations than 10M, and using $0.20 fits well with the $0.50 we're now charging for storage. So my vote would be to use $0.20 and 1M.

Reply:
Ok, SGTM